rqt_controller_manager/rqt_controller_manager/controller_manager.py

```diff
@@ -228,19 +228,19 @@ def _on_ctrl_menu(self, pos):
         # Show context menu
         menu = QMenu(self._widget.ctrl_table_view)
         if ctrl.state == "active":
-            action_deactivate = menu.addAction(self._icons["inactive"], "Deactivate")
-            action_kill = menu.addAction(self._icons["unloaded"], "Deactivate and Unload")
+            action_deactivate = menu.addAction(self._icons["inactive"], "Deactivate (inactive)")
+            action_kill = menu.addAction(self._icons["unloaded"], "Deactivate and Unload (unloaded)")
         elif ctrl.state == "inactive":
-            action_activate = menu.addAction(self._icons["active"], "Activate")
-            action_cleanup = menu.addAction(self._icons["unconfigured"], "Cleanup")
+            action_activate = menu.addAction(self._icons["active"], "Activate (active)")
+            action_cleanup = menu.addAction(self._icons["unconfigured"], "Cleanup (unconfigured)")
```

Comment:
```suggestion
            action_cleanup = menu.addAction(self._icons["unconfigured"], "Unload and load (unconfigured)")
```

As I've mentioned in the issue, I don't have a strong opinion on this "Unload and load" or "cleanup", whether to keep it or remove it. 
If @christophebedard are ok with it, fine for me

Reply:
I am not of a too strong opinion on it atm either, I think getting it is not of extreme importance to the issue in question and we can add it later, I'll go for whatever @christophfroehlich is ok with as well

Reply:
Let's use the technically correct version "Unload and load"

Reply:
work, refer 13a4de2